assets/js/tutorials/trade-in-the-dex.js

```diff
@@ -0,0 +1,278 @@
+// Source for interactive examples in the "Trade in the Decentralized Exchange"
+// tutorial.
+
+// Get Credentials, Connect steps handled by the snippet
+
+$(document).ready(() => {
+
+// Look Up Offers --------------------------------------------------------------
+
+function update_exchange_rate(event) {
+  const tpa = $("#taker-pays-amount-1").val()
+  const tga = $("#taker-gets-amount-1").val()
+  const exchange_rate = BigNumber(tga) / BigNumber(tpa)
+  $("#exchange-rate-1").val(`${exchange_rate}`)
+}
+$("#taker-pays-amount-1").change(update_exchange_rate)
+$("#taker-gets-amount-1").change(update_exchange_rate)
+
+$("#look-up-offers").click( async (event) => {
+  const block = $(event.target).closest(".interactive-block")
+  block.find(".output-area").html("")
+  const address = get_address(event)
+  if (!address) {return}
```

Comment:
Should this return an error?

Reply:
The `get_address()` function (see: [interactive-tutorial.js](https://github.com/XRPLF/xrpl-dev-portal/blob/master/assets/js/interactive-tutorial.js) prints an error to the current interactive block, but it can't cause the parent function to return.